app/server/routes.py

```diff
@@ -98,10 +101,7 @@ async def remove_policy(
 
 
 @router.get("/data")
-async def get_data(dependencies=Depends(TokenBearer())) -> dict:
-    res = r.get(url=settings.OPAL_SERVER_DATA_URL)
-    if res.status_code != 200:
-        raise HTTPException(
-            status_code=res.status_code, detail="Could not retrieve data"
-        )
-    return res.json()
+async def get_data() -> dict:
+    schema_name = "geostore"
+    sql = f"select u.name, g.groupname from {schema_name}.gs_usergroup_members r join {schema_name}.gs_usergroup g on r.group_id = g.id join {schema_name}.gs_user u on r.user_id = u.id;"
+    return {"users": database.get_data(sql=sql)}
```

Comment:
We can put this in the `datasource_database.py` file instead of including it in the routes.

Reply:
Hm alright